src/Lightning.ts

```diff
@@ -72,6 +65,15 @@ export const LightningMixin = (superclass) => class extends superclass {
 
     const expires_at = this.getExpiration(moment()).toDate()
     
+    let lnd: AuthenticatedLnd, pubkey: string
+
+    try {
+      ({ lnd, pubkey } = getActiveLnd())
+    } catch (err) {
+      this.logger.error("no active lnd to create an invoice")
```

Comment:
Probably better to import and throw new CustomError (if you don't want to create a new error class just for this error)

Reply:
yep good point